ollama.go

```diff
@@ -41,7 +41,6 @@ type OllamaMessageCompletionRequestOptions struct {
 	Stop          string  `json:"stop,omitempty"`
 	TfsZ          float32 `json:"tfs_z,omitempty"`
 	NumPredict    int     `json:"num_predict,omitempty"`
-	TopK          int     `json:"top_k,omitempty"`
```

Comment:
why?

Reply:
I was trying to set the `top_k` value for the `Google` integration, and I noticed we weren't configuring any place to put the value from the settings. So I removed all the references of this value from the `Google` implementation, but must have also removed this one by mistake.

Does `mods` currently support setting this value? Because I was thinking about adding it to a new PR.

Reply:
With these latest changes I thin I've normalized the usage of the `topk` option across all apis.